tests/model/test_core.py

```diff
@@ -1761,3 +1762,52 @@ def test_graphviz_call_function(self, var_names, filenames) -> None:
                 figsize=None,
                 dpi=300,
             )
+
+
+class TestModelCopy(unittest.TestCase):
```

Comment:
We use pytest, any reason we need to subclass `unittest`?

Reply:
Oh, I must have made a mistake. I am sorry I am new to using pytest. I will fix that as well!